about/pricing/comparison.md

```diff
@@ -0,0 +1,67 @@
+# Comparing pricing to similar services
+
+This page is a guide to 2i2c's services, our pricing, and how this compares with similar kinds of offerings.
+
+:::{tip}
+The content on this page can be re-used as a part of "price reasonableness and comparisons" forms when completing contracting for communities.
+:::
+
+## How to think about 2i2c's service and pricing
+
+As a non-profit, we choose our prices as a function of our estimated costs.
+Our service entails developing and managing entirely open-source, vendor-agnostic, and community-driven infrastructure that is customized for research and education.
+
+Our aim with 2i2c's service model is to strike a balance between **scalability** and **flexibility**, with the constraints that we operate **transparently** and **collaboratively** with our communities, by runing **open-source** and **community-driven** infrastructure.
+
+We curate and integrate this infrastructure, customize it for use-cases in research and education, and contribute back to the open source communities that underlie the tools we use.
+We use shared configuration and deployment infrastructure so that we can achieve economies of scale in serving many communities, but with an entirely open-source stack that can be customized for their unique needs.
+
+We do not know of any other organizations that offer managed cloud services in this way.
+However, below are three similar kinds of service and product models.
+They are roughly ordered from "most similar" to "least similar" to 2i2c.
+
+## Internal staffing
+
+The most common way for organizations to achieve similar services is to staff their own internal teams.
+2i2c encourages this, as it is aligned with our commitment to open source, vendor-agnostic tools, and the [Right to Replicate your infrastructure](https://2i2c.org/right-to-replicate).
+
+However, hiring and retaining modern cloud engineers is difficult and costly.
+If we assume that an engineer makes `$140,000` in salary, with `30%` benefits, that comes to an annual cost of `$182,000` a year, discounting any other personnel, hiring, and cloud costs.
```

Comment:
I am not sure if you should drop "amounts" here, mostly because it will get out of date pretty soon. Maybe you can reformulate this one in percentage terms?

Reply:
As in % FTE? Maybe we could just say something more vague like `A 100% cloud engineer may cost upwards of $200,000 a year after considering benefits and overhead.`?

Reply:
Damian's feedback is wise. That said, I propose an alternate path for the edits. I suggest we retain the dollar amounts informing the cost structure but include a link or two that points to comparable salaries. The text should reference a date like ("In 2022, the average salary for a Site Reliability Engineer is ....") and a new sentence saying that 2i2c will constantly revisit costs and adjust service pricing. 

A theme I've been exploring that is not widely present in 2i2c's documentation is that "interactive computing should be a public good". No one should have to pay to use the alphabet to communicate. A decade-scale goal for 2i2c is to facilitate the availability of interactive computing to transform the way research and education is carried out. Interactive Computing is a modern analogue of telephone service and will be required for successful participation in democracy.

Reply:
Thank you for putting this together. Having some form of citable documentation on how much development costs is priceless in communicating with administrations. I fully agree with @colliand that including points to comparable salaries in the documentation would even make it better, especially for academic institutions.

Reply:
Thanks for that feedback! We've added that to the doc here: https://github.com/2i2c-org/docs/pull/132/files#diff-6ad2356a04852d0c1b65a04341fe38612743628108ab7c1e43b27ed8db03a754R32 does that look good?